scripts/client.js

```diff
@@ -0,0 +1,25 @@
+import KintoClient from "kinto-client";
+
+
+let client;
```

Comment:
I agree 100% and this is my main gripe with the patch and the previous implementation where it was the same.

The only way I could see having a state-dependent construct of the client would be to store it in the store, which sounds odd. Do you have an idea how we could avoid relying on a singleton in the current architecture? Or do you have another architecture in mind where achieving this would be easy?

Last, could we iterate here? :D